pkg/dao/task_info.go

```diff
@@ -60,6 +60,22 @@ func (t taskInfoDaoImpl) List(
 	return api.TaskInfoCollectionResponse{Data: taskResponses}, totalTasks, nil
 }
 
+func (t taskInfoDaoImpl) IsSnapshotInProgress(repoUUID string) (bool, error) {
```

Comment:
since repoUUID here isn't  a repoConfig, its not tied to an org, so i believe you need to pass in org so its only looking at that org's tasks

Reply:
Oh whoops, yeah